core/kazoo_documents/src/kz_account.erl

```diff
@@ -167,16 +168,30 @@ set_language(JObj, Language) ->
 %% @doc
 %% @end
 %%--------------------------------------------------------------------
--spec timezone(doc()) -> api_binary().
--spec timezone(doc(), Default) -> ne_binary() | Default.
+-spec timezone(ne_binary() | doc()) -> api_binary().
+-spec timezone(ne_binary() | doc(), Default) -> ne_binary() | Default.
+timezone(AccountId)
+  when is_binary(AccountId) ->
+    {'ok', JObj} = fetch(AccountId),
+    timezone(JObj);
 timezone(JObj) ->
     timezone(JObj, 'undefined').
+timezone(AccountId, Default)
+  when is_binary(AccountId) ->
+    {'ok', JObj} = fetch(AccountId),
```

Comment:
@fenollp i tend to agree but in cases where the value may actually be a merge down from hierarchy it hides the complexity of inheritance from whapps.